### PR TITLE
Add one more case to troubleshooting guide

### PR DESCRIPTION
**Pull request by rynowak**
# Description

Found this was missing with a user's help

### CODE REVIEW

File: daprdocs/content/en/dotnet-sdk-docs/dotnet-troubleshooting/dotnet-troubleshooting-pubsub.md [hunk 1]
```diff
@@ -10,12 +10,36 @@ description: Try out .NET virtual actors with this example
...
-## Step 1: Verify endpoint registration
+## Step 1: Turn up the logs
+
+**This is important. Future steps will depend on your ability to see logging output. ASP.NET Core logs almost nothing with the default log settings, so you will need to change it.**
+
+Adjust the logging verbosity to include `Information` logging for ASP.NET Core as described [here](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-5.0#debug-diagnostics). Set the `Microsoft` key to `Information`.
+
+## Step 2: Verify you can recieve traffic from Dapr
+
+1. Start the application as you would normally (`dapr run ...`). Make sure that you're including an `--app-port` argument in the commandline. Dapr needs to know that your application is listening for traffic. Usually this is port 5000 in development.
```
[the remark this answers is not in the document]
Comment (rynowak) [reply daprdocs/content/en/dotnet-sdk-docs/dotnet-troubleshooting/dotnet-troubleshooting-pubsub.md:27]: It's both :)

File: daprdocs/content/en/dotnet-sdk-docs/dotnet-troubleshooting/dotnet-troubleshooting-pubsub.md [hunk 2]
```diff
@@ -139,9 +163,7 @@ Keep this output, as we'll use the `route` information to test the application.
 
 1. Start the application as you would normally (`dapr run ...`).
    
-2. Adjust the logging verbosity to include `Information` logging for ASP.NET Core as described [here](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-5.0#debug-diagnostics). Set the `Microsoft` key to `Information`.
-
-3. Use `curl` at the command line (or another HTTP testing tool) to access one of the routes registered a pub/sub endpoint.
+2. Use `curl` at the command line (or another HTTP testing tool) to access one of the routes registered a pub/sub endpoint.
```
Comment (vinayada1) [inline daprdocs/content/en/dotnet-sdk-docs/dotnet-troubleshooting/dotnet-troubleshooting-pubsub.md:166]: registered "with" a pub/sub endpoint

Comment (rynowak) [reply daprdocs/content/en/dotnet-sdk-docs/dotnet-troubleshooting/dotnet-troubleshooting-pubsub.md:166]:
```suggestion
2. Use `curl` at the command line (or another HTTP testing tool) to access one of the routes registered with a pub/sub endpoint.
```

Comment (rynowak) [reply daprdocs/content/en/dotnet-sdk-docs/dotnet-troubleshooting/dotnet-troubleshooting-pubsub.md]: fixed

**Review by vinayada1 — APPROVED** [review]

**Pull request merged**